pkg/network/tracer/connection/kprobe/perf_batching_test.go

```diff
@@ -73,8 +74,8 @@ func TestGetPendingConns(t *testing.T) {
 
 	batch := new(netebpf.Batch)
 	batch.Id = 0
-	batch.C0.Tup.Pid = 1
-	batch.C1.Tup.Pid = 2
+	batch.C0.Tup.Pid = pidMax + 1
```

Comment:
I'm curious why you switched to using this value? Is it preventing some conflict?

Reply:
Yes, just to avoid any conflicts, although the chance of conflicting on `1` and `2` is pretty remote.